src/main/java/com/mindee/model/geometry/Point.java

```diff
@@ -0,0 +1,10 @@
+package com.mindee.model.geometry;
+
+import lombok.Value;
+
+@Value
+public final class Point {
+  private Double x;
+  private Double y;
+
+}
```

Comment:
no newline at end of file, check your IDE configuration

should follow .editorconfig file: https://github.com/mindee/mindee-api-java/blob/main/.editorconfig#L6